.github/workflows/risk-assessment.yml

```diff
@@ -44,7 +44,7 @@ jobs:
   risk-assessment:
     runs-on: [ self-hosted, small-builder ]
     name: Calculate Risk Score
-    if: ${{ inputs.sra-enabled }}
+    if: ${{ inputs.sra-enabled && !github.event.pull_request.draft }}
```

Comment:
I was thinking to parametrize it again but I think nobody will use it for draft and we don't want to hit limits